.github/workflows/ci.yml

```diff
@@ -26,12 +26,15 @@ jobs:
         needs: [pre-commit]
 
         runs-on: ubuntu-latest
-        timeout-minutes: 10
+        timeout-minutes: 30
 
         strategy:
             fail-fast: false
             matrix:
                 python-version: ['3.8', '3.9', '3.10']
+                default-image:
+                    - ''  # use the application default
+                    - aiidalab/aiidalab-docker-stack:latest
```

Comment:
But `aiidalab/aiidalab-docker-stack` image is built from the old way and only supports 1.6? Am I missing something?

Reply:
A newer version of aiidalab-launch can still run the old image. That is what we are testing here.